packages/teleport/src/Main/Main.tsx

```diff
@@ -119,17 +121,22 @@ export const MainContainer = styled.div`
   display: flex;
   flex: 1;
   position: absolute;
-  min-width: 1000px;
+`;
+
+export const ContentMinWidth = styled.div`
+  // sidebar is 240px + 1px right border
```

Comment:
this is a little confusing
```
// sidebar is 240px + 1px right border
```
I can see that in another place it is
```
// sidebar is 280px + 1px right border
```
Is this intended?

Reply:
Yeah, the sidebar for discover and the main sidebar are different widths

Reply:
Thoughts on moving the value into a CSS variable instead? https://caniuse.com/css-variables